src/routes/_store/storeLite.js

```diff
@@ -19,7 +19,7 @@ export const storeLite = {
 
   set (obj) {
     if (hasLocalStorage) {
-      for (let [key, value] of Object.entries(obj)) {
+      for (const [key, value] of Object.entries(obj)) {
```

Comment:
This might be causing https://github.com/nolanlawson/pinafore/issues/1565

Reply:
See https://stackoverflow.com/a/39045496, it's due to old Firefox